src/main/java/org/graylog/plugins/collector/collectors/rest/models/responses/CollectorRegistrationResponse.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * This file is part of Graylog.
+ *
+ * Graylog is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Graylog is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Graylog.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.graylog.plugins.collector.collectors.rest.models.responses;
+
+
+import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.auto.value.AutoValue;
+
+@AutoValue
+@JsonAutoDetect
+public abstract class CollectorRegistrationResponse {
+    @JsonProperty
+    public abstract int updateInterval();
```

Comment:
Instead of returning the config update like this, I would wrap the settings into a configuration key.

```json
{
  "update_interval": 10,
  "send_status": true
}
```

```json
{
  "configuration": {
    "update_interval": 10,
    "send_status": true
  }
}
```

Thay way we don't have to change the API once we add more information to the response later. (i.e. commands/actions)